src/OrchardCore/OrchardCore.Abstractions/Localization/CultureScope.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Globalization;
+
+namespace OrchardCore.Localization
+{
+    public sealed class CultureScope : IDisposable
+    {
+        private CultureScope(CultureInfo culture)
+        {
+            Culture = culture;
```

Comment:
This is wrong, should be `Culture = CultureInfo.CurrentUICulture` otherwise you will not reset correctly the ui culture in the dispose method